src/test/java/core/basesyntax/MainTest.java

```diff
@@ -0,0 +1,366 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.DataBase;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecord;
+import core.basesyntax.model.FruitStorage;
+import core.basesyntax.service.DataProcessor;
+import core.basesyntax.service.DataProcessorImpl;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.ReportCreatorImpl;
+import core.basesyntax.service.amount.AdditionHandler;
+import core.basesyntax.service.amount.AmountHandler;
+import core.basesyntax.service.amount.SubtractionHandler;
+import core.basesyntax.service.files.FileReader;
+import core.basesyntax.service.files.FileReaderImpl;
+import core.basesyntax.service.files.ReportWriter;
+import core.basesyntax.service.files.ReportWriterImpl;
+import core.basesyntax.service.files.RowParser;
+import core.basesyntax.service.files.RowParserImpl;
+import core.basesyntax.service.strategy.OperationStrategy;
+import core.basesyntax.service.strategy.OperationStrategyImpl;
+import core.basesyntax.service.validation.DataValidator;
+import core.basesyntax.service.validation.DataValidatorImpl;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MainTest {
```

Comment:
Don’t keep all tests in a single class
Create a corresponding test class for each service you test. Do not test logic of whole program in one test class. That’s important for code readability.

Example:

    CsvFileService -> CsvFileServiceTest  
    FruitService -> FruitServiceTest  
    ...

Reply:
Fixed.